fontforgeexe/contextchain.c

```diff
@@ -1748,6 +1748,7 @@ static void RenameClass(struct contextchaindlg *ccd,char *old,char *new,int sect
 	}
     } else {
 	for ( i=0; i<rows; ++i ) {
+		char* end_back = NULL, * end_match = NULL;
 	    char *oldrule = classrules[cols*i+0].u.md_str;
```

Comment:
Spacing needs to be fixed

Reply:
I changed the spaces to tabs, but something is wrong and I don’t know what is wrong.

Reply:
@o-sdn-o For historical reasons much of FontForge is intended with a mixture of spaces and tabs. I suggest copying the whitespace from a line with the same indentation (such as the one below right below) and pasting it.

Reply:
ok, I will do so, and apparently something is wrong with PR checks

Details:
```
continuous-integration/appveyor/p - AppVeyor` build failed
```
```
loading package files...
error: could not open file /var/cache/pacman/pkg/msys2-runtime-3.1.4-1-x86_64.pkg.tar.zst: Child process exited with status 127
error: could not open file /var/cache/pacman/pkg/msys2-runtime-devel-3.1.4-1-x86_64.pkg.tar.zst: Child process exited with status 127
error: failed to commit transaction (cannot open package file)
Errors occurred, no packages were upgraded.
Command exited with code 1
```

Reply:
If you're going to be contributing a lot, in future you should enable visible whitespace in your editor. That's what I do.

![image](https://user-images.githubusercontent.com/838783/82335156-cd35ed00-999d-11ea-9064-86fe7926f918.png)

We might want to recommend this in CONTRIBUTING.md.

Reply:
Ok.

Reply:
Now everything looks consistent
![image](https://user-images.githubusercontent.com/11535558/82361343-2b0d1800-9a24-11ea-9608-f27dde7fe453.png)